### PR TITLE
Update Gateway Docs

### LINKED ISSUE #2116: project gateway  document need to be updated
The document didn't show how to config nginx ingress controller throngh settings and where to find the available settings. We support all the settings in the following document:
https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/

For example:
https://kubesphere.com.cn/forum/d/6071-gzip/2

related document:
https://kubesphere.com.cn/en/docs/cluster-administration/cluster-settings/cluster-gateway/

https://kubesphere.com.cn/docs/project-administration/project-gateway/
https://kubesphere.com.cn/en/docs/project-administration/project-gateway/

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@kubesphere.io>

Fix #2116 

/assign @RolandMa1986 
Please take a look, thanks.

### CODE REVIEW

File: content/en/docs/cluster-administration/cluster-settings/cluster-gateway.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ You need to prepare a user with the `platform-admin` role, for example, `admin`.
...
    - **LoadBalancer**: Access Services with a single IP address through the gateway. The LoadBalancer access mode provides the following configurations:
      - **Tracing**: Turn on the **Tracing** toggle to enable the Tracing feature on KubeSphere. Once it is enabled, check whether an annotation (for example, `nginx.ingress.kubernetes.io/service-upstream: true`) is added for your route when the route is inaccessible. If not, add an annotation to your route.
      - **Load Balancer Provider**: Select a load balancer provider from the drop-down list.
      - **Annotations**: Add annotations to the cluster gateway.
-     - **Configuration Options**: Add key-value pairs to the cluster gateway.
+     - **Configuration Options**: Add key-value pairs to the cluster gateway to provide configurations for system components of NGINX Ingress controller. For more information, see [NGINX Ingress Controller documentation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#configuration-options).
 
    {{< notice info >}}
```
Comment (RolandMa1986) [inline content/en/docs/cluster-administration/cluster-settings/cluster-gateway.md:32]: Looks like duplicated with the **notice** section. No need to change the Cluster Gateway doc.

Comment (Felixnoo) [reply content/en/docs/cluster-administration/cluster-settings/cluster-gateway.md]: Thanks for pointing this out. Fixed.

**Pull request merged**